2021-12-16 20:17:23.361702 | API is disabled on startup
2021-12-16 20:17:23.362703 | Settings loaded!
2021-12-16 20:17:24.989414 | Started VRCA loading process...
2021-12-16 20:17:24.990415 | Propting user to select VRCA...
2021-12-16 20:17:27.260867 | Loaded!
2021-12-16 20:17:28.293598 | Extract VRCA started...
2021-12-16 20:17:28.293598 | Deciding asset URL...
2021-12-16 20:17:28.303609 | Extracting loaded VRCA...
2021-12-16 20:17:28.304610 | Starting extract on new thread...
2021-12-16 20:17:28.305611 | In asset ripper folder
2021-12-16 20:17:28.305611 | Old rip found!
2021-12-16 20:17:28.351653 | Old rip deleted!
2021-12-16 20:17:28.352654 | Returned to base!
2021-12-16 20:17:28.352654 | In asset ripper folder!
2021-12-16 20:18:46.272906 | Extract complete! Cleaning...
2021-12-16 20:18:46.577184 | Cleaned! Opening now...
2021-12-16 20:18:46.580186 | Thread code ended!
2021-12-16 20:18:46.581188 | An error occured during extraction: Traceback (most recent call last):
  File "C:\Users\adamc\Documents\GitHub\A.R.E.S\ARES(rewritten)\ExternalFunctions.py", line 270, in ExtractVRCA
    pymsgbox("Extract complete!")
TypeError: 'module' object is not callable

